include/mlir-tcp/Dialect/IR/TcpOps.td

```diff
@@ -160,6 +160,27 @@ def Tcp_DivFOp : Tcp_BinaryElementwiseOp<"divf", [SameOperandsAndResultElementTy
   let assemblyFormat = "$in1 `,` $in2 attr-dict `:` type($in1) `,` type($in2) `->` type($out)";
 }
 
+def Tcp_DivIOp : Tcp_BinaryElementwiseOp<"divi", [SameOperandsAndResultElementType]> {
+  let summary = "Computes elementwise integer division";
+
+  let description = [{
+    Computes the integer division of `in1` and `in2`.
+  }];
+
+  let arguments = (ins
+    Tcp_IntTensor:$in1,
+    Tcp_IntTensor:$in2,
+    Tcp_SignednessAttr:$signedness,
+    Tcp_RoundingModeAttr:$rounding_mode
```

Comment:
Why do we need a "Ceil" rounding mode here? Are they any cases where that is necessary?

Reply:
I don't think there's a good use-case for the `tcp.diviop` itself to support a ceil rounding mode, given that such a thing does not exist for the `torch.div` op. I added it because there is a natural mapping to the `arith.ceildiviop`. I can remove it.

However, I think the RoundingMode type itself should have ceil in it for completion though.